labely-frontend/src/app/services/labely.service.ts

```diff
@@ -102,4 +102,13 @@ export class LabelyService {
     dwldLink.click();
     document.body.removeChild(dwldLink);
   }
+
+  private removeDupLabelName(labels: Array<Label>): Array<Label> {
```

Comment:
Hello @rjtch thanks for your contribution.
I think it is better to use a 'Set' in 'label.component.ts' directly instead of an array, so that duplicates are not allowed by default